### PR TITLE
MM-19237: convert t.Fatal to use require calls

### PR DESCRIPTION
**Pull request by jesperhansen17**
#### Summary
Converts model/emoji_test.go to use require calls instead of t.Fatal

#### Ticket Link
[MM-19237](https://mattermost.atlassian.net/browse/MM-19237)
Fixes https://github.com/mattermost/mattermost-server/issues/12633

### CODE REVIEW

File: model/emoji_test.go [hunk 1]
```diff
@@ -20,9 +20,8 @@ func TestEmojiIsValid(t *testing.T) {
 		Name:      "name",
 	}
 
-	if err := emoji.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err := emoji.IsValid()
+	require.Nil(t, err)
```
Comment (iomodo) [inline model/emoji_test.go]: We don't need `err` variable
```suggestion
	require.Nil(t, emoji.IsValid())
```

Comment (jesperhansen17) [reply model/emoji_test.go]: Yeah, you are right.

**Comment by iomodo** [comment]
@jesperhansen17 no worries, re-run the jobs, all is clear now :)

**Comment by jespino** [comment]
QA is not needed for this kind of PRs because is only changing the unit tests.

**Comment by hanzei** [comment]
Thanks @jesperhansen17 :+1:

**Pull request merged**